### DISCUSSION TITLE
Sailing Mode Question

### DISCUSSION THREAD

**Discussion by jamesoo777** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#511] [Q&A] [answered]
Hello,

I use my MacBook Pro as a desktop, always plugged in.  Should I enable or disable Sailing Mode?  I already read the feature explanation, but I still need clarification, please.

**Comment by MatthiasKerbl** [comment, marked as answer]
> Hello,
> 
> I use my MacBook Pro as a desktop, always plugged in. Should I enable or disable Sailing Mode? I already read the feature explanation, but I still need clarification, please.

Hi @jamesoo777,

I would enable the Sailing Mode as well. However, depending on your setup, you might not see a big difference.

Cheers,
Matthias
